frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -0,0 +1,651 @@
+import { toPng, toSvg } from "html-to-image";
```

Comment:
this file seems to be really similar to `frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx`, to help with code maintenance and readability, can we see if we could extract common utils shared between `SaveImage` and `SaveExport` to keep the code DRY?

Thank you!

Reply:
Hey Timmy, I actually renamed `SaveImage` to `SaveExport` from [Seve's comment](https://github.com/chanzuckerberg/single-cell-data-portal/pull/4341/files/b3054d43aa023e20de1ab856f2d5c7c544812113#r1137507611)